backend/app/services/device/capability_sync_service.py

```
@@ -0,0 +1,171 @@
+        self, db: Session, user: User, skill_ids: list[int]
+    ) -> list[dict[str, Any]]:
+        if not skill_ids:
+            return []
+
+        rows = (
+            db.query(Kind)
+            .filter(
+                and_(
+                    Kind.id.in_(skill_ids),
+                    Kind.kind == "Skill",
+                    Kind.is_active == True,
```

Ruff (0.15.14)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)